lib/modules/manager/bazel-module/extract.spec.ts

```diff
@@ -54,5 +55,42 @@ describe('modules/manager/bazel-module/extract', () => {
         ],
       });
     });
+
+    it('returns bazel_dep and git_override dependencies', () => {
+      const input = codeBlock`
+        bazel_dep(name = "rules_foo", version = "1.2.3")
+        bazel_dep(name = "rules_bar", version = "1.0.0", dev_dependency = True)
+        git_override(
+          module_name = "rules_foo",
+          remote = "https://github.com/example/rules_foo.git",
+          commit = "850cb49c8649e463b80ef7984e7c744279746170",
+        )
+      `;
+      const result = extractPackageFile(input, 'MODULE.bazel');
+      if (!result) {
+        throw new Error('Expected a result.');
+      }
+      expect(result.deps).toHaveLength(3);
+      expect(result.deps).toContainEqual({
+        datasource: BazelDatasource.id,
+        depType: 'bazel_dep',
+        depName: 'rules_bar',
+        currentValue: '1.0.0',
+      });
+      expect(result.deps).toContainEqual({
+        datasource: BazelDatasource.id,
+        depType: 'bazel_dep',
+        depName: 'rules_foo',
+        currentValue: '1.2.3',
+        skipReason: 'git-dependency',
+      });
+      expect(result.deps).toContainEqual({
+        datasource: GithubTagsDatasource.id,
+        depType: 'git_override',
+        depName: 'rules_foo',
+        currentDigest: '850cb49c8649e463b80ef7984e7c744279746170',
+        packageName: 'example/rules_foo',
+      });
+    });
```

Comment:
combine and use `toEqual` or `toMatchObject`

Reply:
Because the order of the elements is not guaranteed, I ended up using the `arrayContaining()` matcher.